lib/views/index.erb

```diff
@@ -0,0 +1 @@
+<p><%=@text%> is a student at Craft Academy</p> 
```

Comment:
This would read easier if it were `<%= @name %>` (and matching `@name` in the controller), but not a blocker.